qdrant_client/http/api/cluster_api.py

```diff
@@ -46,7 +46,7 @@ def _build_for_cluster_status(
         """
         headers = {}
         return self.api_client.request(
-            type_=m.InlineResponse2002,
+            type_=m.InlineResponse2003,
```

Comment:
```suggestion
            type_=m.InlineResponse203,
```

I think you wanted to do this. Or am I missing something?